packs/ruby/rainbows!/config.yml

```diff
@@ -0,0 +1,16 @@
+name: Rainbows!
```

Comment:
will the `!` in `packs/ruby/rainbows!` cause any issues i wonder?

Reply:
removed in the config, and directory name